packages/okta-react/src/SecureRoute.js

```diff
@@ -20,8 +20,7 @@ const RequireAuth = ({ children }) => {
 
   if(!authState.isAuthenticated) { 
     if(!authState.isPending) { 
-      const fromUri = history.createHref(history.location);
-      authService.login(fromUri);
+      authService.logout();
```

Comment:
Given the pains we've had in tracking this down as desired behavior, perhaps a comment to explain why not login?